test/e2e/ingress/stream.go

```diff
@@ -25,7 +26,7 @@ import (
 	"github.com/apache/apisix-ingress-controller/test/e2e/scaffold"
 )
 
-var _ = ginkgo.Describe("ApisixRoute stream Testing with v2beta1", func() {
+var _ = ginkgo.FDescribe("ApisixRoute stream Testing with v2beta1", func() {
```

Comment:
Please remove the `F` prefix.

Reply:
It has been removed in another commit, please see the latest version of files.